inst/templates/ad_adce.R

```diff
@@ -0,0 +1,157 @@
+# Name: ADCE
+#
+# Label: Clinical Event Analysis Dataset
+#
+# Input: ce, adsl, vs
+library(admiral)
+library(dplyr)
+library(lubridate)
+
+
+# Load source datasets ----
+
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions
+# as needed and assign to the variables below.
+# For illustration purposes read in admiral test data
+
+data("ce")
+data("admiralvaccine_adsl")
+
+
+adsl <- admiralvaccine_adsl
+ce <- ce
```

Comment:
```suggestion
adsl <- convert_blanks_to_na(admiralvaccine_adsl)
ce <- convert_blanks_to_na(ce)
```

R does a lot better with filtering if things in character columns are NA rather than blanks.